overlord/hookstate/hooktest/handler.go

```diff
@@ -68,11 +69,11 @@ func (h *MockHandler) Done() error {
 }
 
 // Error satisfies hookstate.Handler.Error
-func (h *MockHandler) Error(err error) error {
+func (h *MockHandler) Error(err error) (bool, error) {
 	h.Err = err
 	h.ErrorCalled = true
 	if h.ErrorError {
-		return fmt.Errorf("Error failed at user request")
+		return h.IgnoreOriginalErr, fmt.Errorf("Error failed at user request")
```

Comment:
we could always return false here though?

Reply:
Yep. done.